src/Resources/contao/controllers/FrontendIndex.php

```diff
@@ -150,6 +150,12 @@ public function renderPage($pageModel)
 			{
 				$objPage = $objNewPage;
 			}
+
+			// Throw an exception if language was provided but no page was matched
+			elseif (isset($lang) && $lang)
```

Comment:
This should be replaced with `!empty($_GET['lang'])`.

Reply:
You sure? A few lines above you treat the `\Input::get('language')` as the source of truth for language: https://github.com/qzminski/contao-core-bundle/blob/b42099f8575999984b7838dda0a242aaa9462015/src/Resources/contao/controllers/FrontendIndex.php#L137

Reply:
Changed in c035f92c3.